confluent_server/confluent/log.py

```diff
@@ -62,13 +62,21 @@
 import collections
 import confluent.config.configmanager
 import eventlet
-import fcntl
 import json
 import os
 import struct
 import time
 import traceback
 
+try:
+    from fcntl import flock, LOCK_EX, LOCK_UN, LOCK_SH
+except ImportError:
+    flock = lambda file, flag: True
```

Comment:
Just some suggestion. 
How about import msvcrt library if os.name == "nt"?
I think we would better add a common lock util to deal with the difference between posix and nt interface.